python/metric_pipeline_tasks/MatchedCatalogMeasureTasks.py

```diff
@@ -291,11 +291,11 @@ def run(self, matchedCatalogMulti, metric_name, in_id, out_id):
                 if len(finiteEntries) > 0:
                     rmsDistancesAll.append(rmsDistances[finiteEntries])
 
-            rmsDistancesAll = np.array(rmsDistancesAll)
-
             if len(rmsDistancesAll) == 0:
                 return Struct(measurement=Measurement(metric_name, np.nan*u.marcsec))
-            return Struct(measurement=Measurement(metric_name, np.mean(rmsDistancesAll)*u.marcsec))
+            else:
+                rmsDistancesAll = np.array(np.concatenate(rmsDistancesAll))
```

Comment:
Is the `np.array` needed? I had thought that `np.concatenate` returns a `np.array`